jupyterlab_translate/utils.py

```diff
@@ -289,13 +291,65 @@ def get_line(lines: List[str], value: str) -> str:
     return str(line_count)
 
 
+def _extract_schema_strings(
+    schema: dict, schema_lines: list, ref_path: str, prefix: str = ""
+):
+    message_context = "schema"
+
+    to_translate = [
+        "title",
+        "description",
+        "properties/.*/title",
+        "properties/.*/description",
+        "definitions/properties/.*/title",
+        "definitions/properties/.*/description",
```

Comment:
Is this not more proper?

```suggestion
        "definitions/.*/properties/.*/title",
        "definitions/.*/properties/.*/description",
```